doto-frontend/src/components/pages/Calendar/TaskScheduler.js

```diff
@@ -23,17 +23,35 @@ const addTaskToSchedule = (newTask, existingTasks, currDate) => {
 
     // Separate existing tasks with a startDate > current datetime from older tasks (which won't be considered)
```

Comment:
Just for completeness, since you changed the comments below you may want to do it here.